Dockerfile

```diff
@@ -8,6 +8,8 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV PHP_INI_SCAN_DIR="/etc/php/7.2/cli/conf.d:/var/www/src/docker/etc/php"
```

Comment:
Is line 8 not necessary? should it be removed?

Reply:
It is necessary, that's how it finds the xdebug config. Would the path have 7.4 in it now?